src/main/java/nextstep/path/config/finderConfig.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.path.config;
+
+import nextstep.path.service.DijkstraShortestPathService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class finderConfig {
```

Comment:
클래스 이름은 관례에 따라 영어 대문자로 시작하도록 작성해보면 어떨까요?
```suggestion
public class FinderConfig {
```